### ISSUE TITLE
The variable PAGE_NUMBER is not useable in the Custom Thumbnail template

### ISSUE THREAD

**Issue by Bennyz1** [stephanrauh/ngx-extended-pdf-viewer#2240]
With the newer versions 19.5.0 and 19.5.1
When using Custom Thumbnail, in order to print the page number on top of the rendered Thumbnail, using the variable "PAGE_NUMBER".

for example:
(see the #PAGE_NUMBER in the inner div with class "thumbnail-overlay")
```
    <ngx-extended-pdf-viewer 
        [customThumbnail]="richThumbnail">
    </ngx-extended-pdf-viewer>
    <ng-template #richThumbnail>
    <a class="pdf-viewer-template">
        <div class="thumbnail" data-page-number="PAGE_NUMBER">
          <div class="thumbnail-overlay">
              #PAGE_NUMBER
          </div>
        </div>
    </a>
    </ng-template>

```
the actual page number value is not showed, (it just rendered as plain text)
This working perfectly in the older version 19.2.0

**Version info**
- 19.5.1

**Comment by stephanrauh** [comment]
Oops! Yes, you're right, this should work, too... and I even thought I'd covered this, but obviously, I didn't.

**Comment by stephanrauh** [comment]
Your bugfix has landed with version 19.6.1.
